src/app/component/NewsletterSubscription/NewsletterSubscription.style.scss

```diff
@@ -9,12 +9,14 @@
  * @link https://github.com/scandipwa/base-theme
  */
 
-.NewsletterSubscription {
+:root {
     --newsletter-subscription-input-background: var(--input-background-color);
     --newsletter-subscription-input-color: #{$black};
-    --newsletter-subscription-placeholder-color: #9f9c9c;
+    --newsletter-subscription-placeholder-color: #{$default-secondary-dark-color};
```

Comment:
next time use `var(secondary-dark-color)` then the varaible can be changed in the browser via color customization feature.